src/gwt/src/org/rstudio/studio/client/workbench/views/files/Files.java

```diff
@@ -914,7 +914,9 @@ void onSetWorkingDirToFilesPane()
    @Handler
    void onShowFolder()
    {
-      eventBus_.fireEvent(new ShowFolderEvent(currentPath_));
+      String path = server_.resolveAliasedPath(currentPath_);
+      FileSystemItem resolvedCurrentDir = FileSystemItem.createDir(path);
```

Comment:
Does the path need to be created?

Reply:
I agree the name is confusing. The intention is that it follows the pattern for other JavaScript object wrappers in GWT, where they are created with static functions called `create()`. No actual filesystem action is associated with the create method here.

Reply:
(if we felt refactoring for clarity was worthwhile, we could rename these like `fromDirectoryPath()` and `fromFilePath()`)

Reply:
Makes sense after reading the implementation. I'm good either way.